### DISCUSSION TITLE
Anyway to change the font size of the file path/file-name displayed at the top of Brackets?

### DISCUSSION THREAD

**Discussion by m-git-h** [brackets-cont/brackets#244] [General]
Trying to change the size of the font for the path/file-name of the file you are editing, it displays it at the very top of the application.

When the path to the file is too long (in my case all the time) I can't always easily see which location file I'm editing.
If you are like me and are editing same/similar file in many locations this can get confusing

**Comment by abose** [comment]
@supak111 it may not be currently possible in brackets. It should be fixed in phcode.dev likely. 
Can you try it in https://phcode.dev and see if it is a problem there too?
